ios/browser/api/brave_shields/adblock_engine.h

```diff
@@ -80,30 +80,13 @@ OBJC_EXPORT
 - (NSString*)cosmeticResourcesForURL:(NSString*)url
     NS_SWIFT_NAME(cosmeticResourcesForURL(_:));
 
-/// Returns a stylesheet containing all generic cosmetic rules that begin with
-/// any of the provided class and id selectors
-///
-/// The leading '.' or '#' character should not be provided
-- (NSString*)
-    stylesheetForCosmeticRulesIncludingClasses:(NSArray<NSString*>*)classes
```

Comment:
Is there a replacement for this method at all?

Reply:
not in my commit. i did a quick github search of brave-ios but didn't find any usages. github search sometimes fails to find things, so i suppose there is a usage? 

if so, i will need some assistance from an iOS dev to implement a replacement

Reply:
it was added in preparation for a PR that @cuba is actively working on, perhaps he can give you a pointer